### PR TITLE
UI: Added Unit test and localisation support for profilerTab component [Part-2]

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Added Unit test and localisation support for profilerTab component
#8074
#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str: string) => str,
+    };
+  },
+}));
+
+describe('Test ProfilerTab component', () => {
+  beforeEach(() => cleanup());
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx:59]: We don't need this one as it's mentioned globally [here](https://github.com/open-metadata/OpenMetadata/blob/c4b7432c073f080fbd668c583f9a6a170ecb96b2/openmetadata-ui/src/main/resources/ui/jest.config.js#L45)

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    );
+
+    expect(pageContainer).toBeInTheDocument();
+    expect(description).toBeInTheDocument();
+    expect(dataTypeContainer).toBeInTheDocument();
+    expect(ProfilerSummaryCards).toHaveLength(2);
+    expect(ProfilerDetailsCards).toHaveLength(4);
+  });
+
+  it('ProfilerTab component should render properly with empty data', async () => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx:85]: We do not have any interaction for this component?

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx:59]: Oh ok, Thanks for highlighting 🙏🏻

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx:85]: This component only displays the data and doesn't have any interactions

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**